src/KintoID.sol

```diff
@@ -57,15 +57,21 @@ contract KintoID is
     /// state-changing operation, so as to prevent replay attacks, i.e. the reuse of a signature.
     mapping(address => uint256) public override nonces;
 
-    bytes32 public domainSeparator;
+    bytes32 public override domainSeparator;
```

Comment:
why do we need all these vars to be override?

Reply:
to have them in the interface